### ISSUE TITLE
some problems

### ISSUE THREAD

**Issue by WLXM0922** [jake3991/DRACo-SLAM#4]
Hello, thank you for your contribution. I encountered some problems when I reappeared your work recently, and I look forward to your reply.
First of all, in the dead_reckoning.py file, it seems that you don't have the initialization parameter self.rov_id, which makes it impossible to correctly publish the transformation from odom coordinate system to base_link coordinate system of each robot.
Next，when I run roslaunch draco_slam slam.launch, I open a new terminal to run roabag play sample_data_three_robot.bag, and it displays "unknown frame world" in rviz.
However, the launch file of each rov has the following definition: <node pkg="tf" name="map_to_world_tf_publisher" type="static_transform_publisher" args="0 55 0 0 0 3.14159 world rov_one_map 100"/>.
Why is it wrong to report "unknown frame world"? I would appreciate it if you could give me some advice.

**Comment by ConorGagliardi** [comment]
Not sure if you're still having this error, but if you use "rosbag play --clock sample_data_three_robot.bag" it should work.

**Comment by WLXM0922** [comment]
> Not sure if you're still having this error, but if you use "rosbag play --clock sample_data_three_robot.bag" it should work.

Yeah, I have successfully run through this demo using rosbag play --clock. Thank you very much for your reply.
